.github/workflows/unittest.yml

```diff
@@ -34,6 +34,10 @@ jobs:
             python-version: "3.8"
           - tf-version: 2.12.0
             python-version: "3.11"
+          - tf-version: 2.13.0
+            python-version: "3.8"
+          - tf-version: 2.13.0
+            python-version: "3.11"
```

Comment:
Just checking, is this on purpose that we want to test it for two Python versions?

Reply:
This is mainly to match what we've been doing for other TF versions where we test against the oldest and newest Python version. But since 2.13 doesn't support Python 3.7 I've added this explicitly here matching what we do for TF 2.13.

I'm also happy to drop Python 3.7 since it's EOL since last week, but we probably should do that in a separate PR.